staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/metrics.go

```diff
@@ -56,6 +56,17 @@ var (
 		},
 	)
 
+	lastRequestArrivalTime = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "last_request_arrival_time_seconds",
+			Help:           "Last time a request arrived.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"transformation_type"},
```

Comment:
What are the possible values for this? (I feel like you've told this to me before but I've forgotten since)

Reply:
This is bound to two values: fromStorage and toStorage.